setup.py

```diff
@@ -11,9 +11,26 @@
 
 dependencies = [
     "cloudquery-plugin-pb==0.0.14",
-    "pyarrow==12.0.1",
+    "exceptiongroup==1.1.2",
+    "black==23.7.0",
+    "grpcio==1.56.2",
+    "grpcio-tools==1.56.2",
+    "iniconfig==2.0.0",
     "Jinja2==3.1.2",
+    "MarkupSafe==2.1.3",
+    "numpy==1.25.1",
+    "packaging==23.1",
+    "pandas==2.0.3",
+    "pluggy==1.2.0",
+    "protobuf==4.23.4",
+    "pyarrow==12.0.1",
+    "pytest==7.4.0",
+    "python-dateutil==2.8.2",
+    "pytz==2023.3",
+    "six==1.16.0",
     "structlog==23.1.0",
+    "tomli==2.0.1",
+    "tzdata==2023.3",
```

Comment:
I've moved all the requirements into setup.py, and changed requirements.txt to `.`, so that they are the same. We can change this again later, but this should ensure that the package and dev requirements are the same for now. Only thing is that we probably don't need `black` here